spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_graph_target_vm_spec.rb

```diff
@@ -7,6 +7,12 @@
       @ovirt_service = ManageIQ::Providers::Redhat::InfraManager::OvirtServices::Strategies::V4
       allow_any_instance_of(@ovirt_service)
         .to receive(:collect_external_network_providers).and_return(load_response_mock_for('external_network_providers'))
+      @collector = ManageIQ::Providers::Redhat::Inventory::Collector
+      allow_any_instance_of(@collector)
```

Comment:
Please have at least one spec per regular and graph refresh which actually collects `vnic_profiles`. We need to make sure we update the inventory correctly.

Reply:
ovirt_sdk_refresh_recording.yml has vnic profile. Both refresher_4_async_graph_spec.rb and refresher_4_async_spec.rb use it. Updated the tests to make sure the lan was set correctly.